system-monitor@paradoxxx.zero.gmail.com/gpu_usage.sh

```diff
@@ -30,8 +30,12 @@ checkcommand()
 if checkcommand nvidia-smi; then
 	nvidia-smi -i 0 --query-gpu=memory.total,memory.used,utilization.gpu --format=csv,noheader,nounits | sed 's%, %\n%g'
 elif checkcommand glxinfo; then
-	TOTALVRAM=$(glxinfo | grep -A2 -i GL_NVX_gpu_memory_info | grep -E -i "dedicated" | cut -f2- -d ':' | gawk '{print $1}')
-	AVAILVRAM=$(glxinfo | grep -A4 -i GL_NVX_gpu_memory_info | grep -E -i "available dedicated" | cut -f2- -d ':' | gawk '{print $1}')
+	TOTALVRAM=$(glxinfo | grep -A2 -i GL_NVX_gpu_memory_info | grep -E -i "dedicated")
+	TOTALVRAM=${TOTALVRAM##*:[[:blank:]]}
+	TOTALVRAM=${TOTALVRAM%%[[:blank:]]MB*}
+	AVAILVRAM=$(glxinfo | grep -A4 -i GL_NVX_gpu_memory_info | grep -E -i "available dedicated")
+	AVAILVRAM=${AVAILVRAM##*:[[:blank:]]}
+	AVAILVRAM=${AVAILVRAM%%[[:blank:]]MB*}
```

Comment:
These 2 lines could also be replaced by one:
`AVAILVRAM=${AVAILVRAM//[^[:digit:]]/}`

Reply:
Yes, but that is bash-specific syntax and not POSIX, which would have to be replaced with these two lines in order to get e.g. https://github.com/paradoxxxzero/gnome-shell-system-monitor-applet/pull/591 in place. And there is no need to use bash when the job can be done with 10% of the computing resources and footprint with a POSIX shell such as dash (default /bin/sh shell in Debian-based distributions since a couple of releases back).